library/psa_crypto.c

```diff
@@ -7907,6 +7909,9 @@ psa_status_t psa_pake_input(
 {
     psa_status_t status = PSA_ERROR_CORRUPTION_DETECTED;
     psa_crypto_driver_pake_step_t driver_step = PSA_JPAKE_STEP_INVALID;
+    const size_t max_input_length = (size_t) PSA_PAKE_INPUT_SIZE(operation->alg,
```

Comment:
Is it supported for the user or anyone else to change the value of `PSA_PAKE_INPUT_MAX_SIZE`? If so, could we end up with a situation where `PSA_PAKE_INPUT_SIZE()` gives a larger number than `PSA_PAKE_INPUT_MAX_SIZE`?

Reply:
I think users can't change these values. The format for JPAKE and supported primitive is constant and we allow only input of lengths `65` or `32` bytes.
The `PSA_PAKE_INPUT_MAX_SIZE` is the biggest value (`65`).

When we add support for other pake algorithms then `PSA_PAKE_INPUT_SIZE` will have to be adapted for the new algorithm. If input size for the new algorithm is greater than `65`, then also `PSA_PAKE_INPUT_MAX_SIZE` has to be adapted accordingly.